instagram-app/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import '../SearchBar/SearchBar.css';
+import instaLogo from '../../Images/instaLogo.jpg';
+import instagramTypography from '../../Images/instagramTypography.jpg';
+import verticalBar from '../../Images/verticalBar.png';
+
+const SearchBar = () => {
+    return (
+        <div className='SearchBar'>
+            <img src={instaLogo} alt="instaLogo"></img>|
+            <img src={verticalBar} alt="verticalBar"></img>
+            <img src={instagramTypography} alt="instagramTypo"></img>
+
+            <input></input>
```

Comment:
Always make your `input` inside of a `form` element. It works either way, but firstly, it's for semantics, and secondly, there are some features your inputs won't have, unless they are wrapped inside of a form element